Tools/jit/_targets.py

```diff
@@ -145,41 +137,7 @@ async def _compile(
             "-std=c11",
             *self.args,
         ]
```

Comment:
We can combine these. Then you can delete the following line, and just use `args` instead of `args_o` in the `_llvm.run` call.

```suggestion
            "-o",
            f"{o}",
            f"{c}",
            *self.args,
        ]
```